packages/gatsby/src/services/initialize.ts

```diff
@@ -336,25 +328,30 @@ module.exports = async (args: BootstrapArgs) => {
     return undefined
   }
 
-  const ssrPlugins = _.filter(
-    flattenedPlugins.map(plugin => {
+  interface IPluginResolution {
+    resolve: string
+    options: IPluginInfoOptions
+  }
```

Comment:
I think interfaces/type definitions should be at the top of a file. Should we add a lint for this?

Reply:
Done